windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -225,6 +239,11 @@
   <!-- Introduced: 9.0.420.0 -->
   <string name="S_Languages_Install" comment="Add language profile">Add language</string>
 
+  <!-- Context: Configuration Dialog - Captions -->
+  <!-- String Type: PlainText -->
+  <!-- Introduced: 16.0.47 -->
+  <string name="S_Languages_Modify" comment="Add or Remove languages for keyboard layout">Add/remove language</string>

```

Comment:
Is this a dialog title? Then it would be helpful to add that to the comment (assuming the comment is what's visible on Crowdin), e.g.

```suggestion
  <string name="S_Languages_Modify" comment="Title for dialog to add or remove languages for keyboard layout">Add/remove language</string>

```

Reply:
Believe this is the button label?

Reply:
It is used for the button and the Title of the dialog the button opens. Yes, localisation context matters. Should I create one for the button and one for the Title? Note I also am adding the Ellipsis by concatenation. I am wondering if the translators would prefer to have that information also?

Reply:
Wondering - does adding the ellipsis by concatenation work for RTL languages, or do they need an extra whatever added so that it doesn't show "in front of" the text? If that's the case then it would make sense to have two separate strings, otherwise we can go with one, but I'd add the info that it's used for both button and title in the comment.

Reply:
The ellipsis is part of the translatable string and should not be concatenated. There should be two strings -- one for the button, and one for the title, because they may well translate differently in some languages.